src/Repository/CaseEntityRepository.php

```diff
@@ -202,13 +193,50 @@ private function updateQueryBuilderWithBoardFinishStatuses(QueryBuilder $qb): Qu
             $statusDQLVariable = 'board_finish_status_'.$count;
             $boardDQLVariable = 'board_'.$count;
 
-            $qb->orWhere('c.currentPlace = :'.$statusDQLVariable.' AND c.board = :'.$boardDQLVariable)
+            if ($getActive) {
+                $boardExpression->add($qb->expr()->andX(
+                    $qb->expr()->neq('c.currentPlace', ':'.$statusDQLVariable),
+                    $qb->expr()->eq('c.board', ':'.$boardDQLVariable),
+                ));
+            } else {
+                $boardExpression->add($qb->expr()->andX(
+                    $qb->expr()->eq('c.currentPlace', ':'.$statusDQLVariable),
+                    $qb->expr()->eq('c.board', ':'.$boardDQLVariable),
+                ));
+            }
```

Comment:
Could be written as

```php
$boardExpression->add($qb->expr()->andX(
  $getActive
    ? $qb->expr()->neq('c.currentPlace', ':'.$statusDQLVariable)
    : $qb->expr()->eq('c.currentPlace', ':'.$statusDQLVariable),
  $qb->expr()->eq('c.board', ':'.$boardDQLVariable),
));
```

which may make the subtle difference between the two conditions easier to read.

Reply:
Fixed in 332d773517f7a733edd56475bf67262300969905.